python/tidy/servo_tidy/tidy.py

```diff
@@ -389,6 +389,7 @@ def check_rust(file_name, lines):
             # No benefit over using &str
             (r": &String", "use &str instead of &String", no_filter),
             (r"^&&", "operators should go at the end of the first line", no_filter),
+            (r"\{[A-Za-z]+\};$", "use statement contains parenthesis for single import", lambda match, line: line.startswith('use ')),
```

Comment:
This should probably contain numbers too. It won't match `Size2D` (for example), right?